<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-0F765RS2" ignored="4" name="Command line test" tests="4" failures="1" timestamp="2020-11-13T00:40:56 IST" time="2.15" errors="0">
  <testcase name="validateCreateCustomerApiWithValidateSecretKey" time="0.001" classname="com.smc.APITesting.testcases.CreateCustomerTest"/>
  <testcase name="validateCreateCustomerApiWithinValidateSecretKey" time="0.0" classname="com.smc.APITesting.testcases.CreateCustomerTest"/>
  <testcase name="CreateUsr" time="1.899" classname="com.smc.APITesting.testcases.TestCreateUser">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at io.restassured.path.json.JsonPath.getString(JsonPath.java:352)
at com.smc.APITesting.testcases.TestCreateUser.CreateUsr(TestCreateUser.java:39)
at java.util.ArrayList.forEach(ArrayList.java:1259)
Caused by: java.lang.IllegalArgumentException: The JSON input text should neither be null nor empty.
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:80)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 43 more
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- CreateUsr -->
  <testcase name="getUser" time="0.216" classname="com.smc.APITesting.testcases.TestCreateUser"/>
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.smc.APIFramwork.setUp.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.smc.APIFramwork.setUp.BaseTest">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase name="@BeforeSuite setUp" time="0.0" classname="com.smc.APIFramwork.setUp.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
  <testcase name="@BeforeSuite setUp" time="0.0" classname="com.smc.APIFramwork.setUp.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite setUp -->
</testsuite> <!-- Command line test -->
